securedrop/tests/functional/journalist_navigation_steps.py

```diff
@@ -17,6 +18,9 @@ def get_journalist_usernames():
     yield "dellsberg"
     yield "jpb"
     yield "bassel"
+    while True:
+        num = random.randint(1000,1000000)
+        yield "journalist" + str(num)
```

Comment:
@kushaldas Can you add a comment stating _why_ we're doing this? The commit message indicates that we _can_, but is unclear in terms of the motivation. Also:

* the comment above the function is now quite inaccurate (states that we're adding _two_ usernames :slightly_smiling_face:)
* flake8 violations in the diff presented

There are a few flake8 violations in the test files, but we can clean up the test prior to merge into develop.